appveyor.yml

```diff
@@ -0,0 +1,25 @@
+---
+# http://www.appveyor.com/docs/appveyor-yml
+
+# Build version format
+version: "{build}"
+
+# Do not build on tags (GitHub only)
+skip_tags: true
+
+environment:
+  nodejs_version: '4'
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install
+
+test_script:
+  - node --version
+  - npm --version
+  - npm run test:mocha
```

Comment:
True, we lose reports if one fails (as Travis CI does), but the whole point of this is to make sure `npm install` and `npm test` do not break for Windows...

Done in c886657e6b1333ac10c269ada680e3b3316a96e2.